server/api/account.ts

```diff
@@ -0,0 +1,61 @@
+import express from 'express';
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+require('dotenv').config()
+
+const { getPool, createCustomer, getCustomerById} = require('../db');
+const pool = getPool();
+
+const saltRounds = 10;
+
+const accessTokenSecret = process.env.ACCESS_TOKEN_SECRET;
```

Comment:
Same - const string should be all caps snake
```suggestion
const { ACCESS_TOKEN_SECRET }= process.env.ACCESS_TOKEN_SECRET;
```

Reply:
How is destructuring working here? Isn't `ACCESS_TOKEN_SECRET` a string?

Reply:
Oh sorry I meant to delete it from the part after the assignment.
```
const { ACCESS_TOKEN_SECRET }= process.env;
```